documentation/src/main/asciidoc/configuration.asciidoc

```diff
@@ -1427,21 +1429,67 @@ level deployment, in META-INF (or WEB-INF for web deployments).
 
 ==== Update and activate latest Hibernate Search version in WildFly
 
-*Temporarily disabled:*
+You can also download the latest Hibernate Search provided module and install it. This is often the
+best approach as you will benefit from all the latest improvements of
+Hibernate Search. Because of the modular design in WildFly, these additional modules can
+coexist with the embedded modules and won't affect any other application, unless you
+explicitly reconfigure it to use the newer module.
+
+You can download the latest pre-packaged Hibernate Search modules from
+http://sourceforge.net/projects/hibernate/files/hibernate-search/{hibernateSearchVersion}/hibernate-search-modules-{hibernateSearchVersion}-wildfly-10-dist.zip/download[Sourceforge].
+As a convenience these zip files are also distributed as Maven artifacts:
+link:$$https://repository.jboss.org/nexus/index.html#nexus-search;gav~org.hibernate~hibernate-search-modules~{hibernateSearchVersion}~~$$[org.hibernate:hibernate-search-modules-{hibernateSearchVersion}-wildfly-10-dist:zip].
+
+Unpack the modules in your WildFly `modules` directory: this will create modules for Hibernate Search and Apache Lucene.
+The Hibernate Search modules are:
+
+* _org.hibernate.search.orm_, for users of Hibernate Search with Hibernate; this will transitively include Hibernate ORM.
+* _org.hibernate.search.engine_, for projects depending on the internal indexing engine that don't require other dependencies to Hibernate.
+* _org.hibernate.search.backend-jms_, in case you want to use the JMS backend described in <<search-architecture-jms, JMS Architecture>>.
```

Comment:
People don't need to know about those, it will just work :)
Only `backend-jms` requires an explicit import.